oe_theme.theme

```diff
@@ -66,6 +66,14 @@ function oe_theme_preprocess_paragraph__oe_links_block(&$variables) {
   }
 }
 
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_theme_preprocess_paragraph__oe_quote(&$variables) {
+	// Temporary solution to avoid render issue with ECL component that expecting a string.
+  $variables['quoteBody'] = $variables['content']['field_oe_text_long'][0]['#text'];
```

Comment:
Preprocess is not necessary, we can just render `content.field_oe_text_long` in the template.

Reply:
@ademarco I do agree with you, however, the preprocessor was just for testing while we were solving the ECL rending issue when the data is an array type.

Now that we are using the plain text I will test and the preprocessor will be removed.